docs/wcs/index.rst

```diff
@@ -235,6 +235,29 @@ ability to use the :class:`~astropy.wcs.WCS` to define projections in
 Matplotlib. More information on installing and using WCSAxes can be found `here
 <http://wcsaxes.readthedocs.org>`__.
 
+.. plot::
+    :include-source:
+
+    import numpy as np
```

Comment:
You're right. I removed it at one point but it got caught in a copy/paste. I'll remove it.